tests/jvm/src/test/resources/metac.expect

```diff
@@ -1065,9 +1065,8 @@ Occurrences:
 [42:21..42:25): head => scala/collection/IterableLike#head().
 
 Synthetics:
-[39:9..39:9):  => *[Unit]
-  [0:0..0:1): * => _star_.
-  [0:2..0:6): Unit => scala/Unit#
+[39:2..39:9): orig(locally) => orig(locally)[Unit]
```

Comment:
I think that we should optimize for the common case and print `*` instead of `orig(<the original tree on the lhs>)`.

Reply:
I would like to push against this, it is extremely helpful to see the full context when looking at synthetics, it was invaluable for me to debug for statements and I think it would be useful for users of metap.

Reply:
I should've been more precise in this formulation. 

Having parts of the original tree wrapped in `orig(...)` is super helpful. That's one of the very important features of this PR - both datamodel-wise and prettyprinting-wise.

However, when the original tree is used verbatim in the synthetic, this introduces extra verbosity that I'm not sure is worth the cost. For example, currently:

```
[13:14..13:36): #:: 2 #:: Stream.empty => orig(#:: 2 #:: Stream.empty)[Int]
  Int => scala/Int#
```

Here the right-hand side basically duplicates whatever is written on the left-hand side, completely dominating the otherwise pretty small synthetic. Compare this with the proposed:

```
[13:14..13:36): #:: 2 #:: Stream.empty => *[Int]
  Int => scala/Int#
```